components/wishlist/WishlistButton/WishlistButton.tsx

```diff
@@ -26,15 +26,15 @@ const WishlistButton: FC<Props> = ({
   const { data: customer } = useCustomer()
   const { openModal, setModalView } = useUI()
   const [loading, setLoading] = useState(false)
-
```

Comment:
Please remove all changes, including spaces, to leave history untouched

Reply:
It's a bit hard to see changes like spaces made outside the core since we merged with `main`. Maybe because I didn't make those changes from my end. @acasazza can you take a look from yours?